api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1497,6 +1497,10 @@ type HostedClusterStatus struct {
 	// Platform contains platform-specific status of the HostedCluster
 	// +optional
 	Platform *PlatformStatus `json:"platform,omitempty"`
+
+	// OldestKubeletVersion tracks the oldest kubelet version in a hosted cluster
+	// +openshift:enable:FeatureGate=MinimumKubeletVersion
```

Comment:
for this to take effect you'll want to set disable/enable the v1beta1/featuregates, then make update